doc/select.md

```diff
@@ -0,0 +1,204 @@
+# Select examples
+
+## Filtering
+
+The second parameter passed to ``crud.select`` is an array of conditions. Below are examples of filtering data using these conditions. 
+
+### Getting full space
+
+To get a full space without filtering, you need to pass ``nil`` as a condition.
+
+**Example:**
+
+```lua
+crud.select('customers', nil)
+---
+- metadata:
+  - {'name': 'id', 'type': 'unsigned'}
+  - {'name': 'bucked_id', 'type': 'unsigned'}
+  - {'name': 'name', 'type': 'string'}
+  - {'name': 'surname', 'type': 'string'}
+  - {'name': 'age', 'type': 'number'}
+  rows:
+  - [1, 635, 'John', 'Adams', 65]
+  - [2, 2364, 'John', 'Sidorov', 28]
+  - [3, 6517, 'Ronald', 'Dump', 77]
+  - [4, 563, 'Sergey', 'Lee', 21]
+  - [5, 2313, 'Tatyana', 'May', 20]
+...
+```
+
+### Select using index
+
+Let's say we have a ``age`` index. Example below gets a list of ``customers`` over 30 years old.
+
+**Example:**
+
+```lua
+res, err = crud.select('customers', {{'>=', 'age', 30}})
+res.rows
+---
+- - [1, 635, 'John', 'Adams', 65]
+  - [3, 6517, 'Ronald', 'Dump', 77]
+...
+```
+
+### Select using composite index
+
+Suppose we have a composite index consisting of the ``name`` and ``surname`` fields. See example of select queries using such a composited index below.
+
+**Example**:
+
+```lua
+res, err = crud.select('customers', {{'==', 'full_name', {"John", "Sidorov"}}})
+res.rows
+---
+- - [1, 2364, 'John', 'Sidorov', 28]
+...
+```
+
+### Select using partial key 
+
+Alternatively, you can use a partial key for a composite index.
+
+**Example**:
+
+```lua
+res, err = crud.select('customers', {{'==', 'full_name', "John"}})
+res.rows
+---
+- - [1, 2364, 'John', 'Sidorov', 28]
+  - [2, 635, 'John', 'Adams', 65]
+...
+```
+
+### Select using non-indexed field
+
+You can also make a selection using a non-indexed field.
+
+**Example:**
+
+```lua
+res, err = crud.select('customers', {{'>=', 'id', 3}})
+res.rows
+---
+- - [3, 6517, 'Ronald', 'Dump', 77]
+  - [4, 563, 'Sergey', 'Lee', 21]
+  - [5, 2313, 'Tatyana', 'May', 20]
+...
+```
+
+**Note:** in this case full scan is performed.
+
+Note that the search condition for the indexed field must be placed first to avoi a full scan.
+
+**Example:**
+
+```lua
+res, err = crud.select('customers', {{'>=', 'id', 3}, {'>=', 'age', 30}})
+res.rows
+---
+- - [3, 6517, 'Ronald', 'Dump', 77]
+...
+```
+
+In this case, a full scan will be performed, since non-indexed field is placed first in search conditions. Example below shows how you can avoid a full scan.
+
+**Example:**
+
+```lua
+res, err = crud.select('customers', {{'>=', 'age', 30}, {'>=', 'id', 3}})
+res.rows
+---
+- - [3, 6517, 'Ronald', 'Dump', 77]
+...
+```
+
+## Pagination
+
+The third (but optional) parameter in ``crud.select`` is array of [``options``](https://github.com/tarantool/crud#select). With this parameter, we can implement pagination. 
```

Comment:
You can specify pagination (and other) options using `opts` argument.

Your description is good, but too long and detailed. `select` API is described in README, this document is used to show how it can be used.